src/datasources/alerts-api/configuration/alerts-api.configuration.module.ts

```diff
@@ -0,0 +1,23 @@
+import { DynamicModule, Module } from '@nestjs/common';
+import { IConfigurationService } from '@/config/configuration.service.interface';
+import { ConfigFactory } from '@nestjs/config/dist/interfaces/config-factory.interface';
+import { ConfigModule } from '@nestjs/config';
+import { NestConfigurationService } from '@/config/nest.configuration.service';
+import alertsApiConfiguration from '@/datasources/alerts-api/configuration/alerts-api.configuration';
+
+@Module({})
+export class AlertsApiConfigurationModule {
+  static register(configFactory: ConfigFactory): DynamicModule {
+    return {
+      module: AlertsApiConfigurationModule,
+      imports: [ConfigModule.forFeature(configFactory)],
+      providers: [
+        { provide: IConfigurationService, useClass: NestConfigurationService },
+      ],
+      exports: [IConfigurationService],
+    };
+  }
+}
+
+export const alertsApiConfigurationModule =
```

Comment:
Nit: I was initially confused regarding the naming of this (starting with a lowercase character). What do you think about adjusting to to something like `RegisteredAlertsApiConfigurationModule`?

Reply:
I'd be in favor of using `SCREAMING_SNAKE_CASE` here 👍

Reply:
I also agree.